src/plugins/MonkeyPatched.ts

```diff
@@ -15,8 +15,17 @@ export abstract class MonkeyPatched<
     Nodule extends object,
     FieldName extends keyof Nodule
 > extends InternalPlugin {
-    public enable = this.patch.bind(this, true);
-    public disable = this.patch.bind(this, false);
+    constructor(pluginId: string) {
+        super(pluginId);
+    }
+
+    enable() {
```

Comment:
Enable / disable were refactored from properties to methods. This way they can be overriden while still having access to the originale methods via `super.enable/disable`